.NET/Microsoft.Recognizers.Definitions.Common/German/NumbersDefinitions.cs

```diff
@@ -43,11 +43,12 @@ public static class NumbersDefinitions
       public static readonly string AllIntRegexWithLocks = $@"((?<=\b){AllIntRegex}(?=\b))";
       public static readonly string AllIntRegexWithDozenSuffixLocks = $@"(?<=\b)(((ein\s+)?halbes\s+dutzend)|({AllIntRegex}\s+dutzend(e)?))(?=\b)";
       public const string RoundNumberOrdinalRegex = @"(hundertst(er|es|en|el|e)?|tausendst(er|es|en|el|e)?|millionst(er|es|en|el|e)?|milliardst(er|es|en|el|e)?|billionst(er|es|en|el|e)?)";
-      public const string RelativeOrdinalRegex = @"(?<relativeOrdinal>(ante)?penultimate|letzter|nächster|vorheriger|aktueller)";
+      public const string RelativeOrdinalRegex = @"(?<relativeOrdinal>letzte[nr]?|nächste[nr]?|vorherige[nr]?|aktuelle[nr]?|jetzige[nr]?|vorletzte[nr]?|zuletzt|früher)";

```

Comment:
Shouldn't we also cover drittletzter?

Reply:
vorletzte[nr]? and letzte[nr]? can also be combined in a simplified regex.

Reply:
added drittletzter and simplified regex